src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-intro.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Introduction to OpenTelemetry Collector with New Relic
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: The OpenTelemetry collector is a central tool to collect, process, and export your telemetry to New Relic and beyond.
+---
+
+The OpenTelemetry collector is a component you can implement to receive and process your telemetry data before exporting it to New Relic or another observability back-end. Instead of sending data from your apps directly to New Relic, you can first route it to a central collector. This shifts the overhead of managing telemetry away from your apps and moves it into a single component.
+
+![Diagram showing OpenTelemetry using the OpenTelemetry Collector and New Relic's OTLP endpoint.](./images/native_otlp_with_collector.png "Diagram of OTLP with collector")
+<figcaption>
+Here's an example of the collector as a gateway, but you can also set up the collector as an agent that runs with your application or runs on the same host.
+</figcaption>
+
+## When to use a collector? [#when]
+
+In many cases, the collector is an optional component, but you may need it for a variety of reasons:
+
+    * Gather metrics about your infrastructure, for example, capturing CPU usage on the servers hosting your apps.
+    * Ingest signals in a variety of formats (Jaeger, Prometheus, Fluent Bit, etc.).
+    * Send data to multiple observability back-ends.
+    * Transform metrics or change the names of spans before exporting them to an observability back-end.
+    * Offload the work of sending data over the network when you're running resource-intensive services.
+    * Enrich telemetry (metrics, traces and logs) with additional metadata.
+
+## Learn about how the collector works [#learn]
+
+To get an overview of the processes inside the collector, as well as deployment options, see the following OpenTelemetry docs:
+
+  * [Data collection](https://opentelemetry.io/docs/concepts/data-collection/)
```

Comment:
Would it be worth flagging these as external links?

Reply:
@paperclypse, thanks for this tip! I'll catch it in a fast-follow!